must-gather/pkg/cli.go

```diff
@@ -0,0 +1,357 @@
+package pkg
+
+import (
+	"fmt"
+	"slices"
+	"strconv"
+	"strings"
+	"time"
+
+	volumesnapshotv1 "github.com/kubernetes-csi/external-snapshotter/client/v8/apis/volumesnapshot/v1"
+	nac1alpha1 "github.com/migtools/oadp-non-admin/api/v1alpha1"
+	openshiftconfigv1 "github.com/openshift/api/config/v1"
+	oadpv1alpha1 "github.com/openshift/oadp-operator/api/v1alpha1"
+	ocadminspect "github.com/openshift/oc/pkg/cli/admin/inspect"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/spf13/cobra"
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	velerov2alpha1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v2alpha1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	"github.com/openshift/oadp-operator/must-gather/pkg/gather"
+	"github.com/openshift/oadp-operator/must-gather/pkg/templates"
+)
+
+const (
+	mustGatherVersion = "master-branch"
```

Comment:
this needs to change to `1.5.0`

better to change after this one merges in a separate PR?

Reply:
might as well catch this in follow-on PR along with https://github.com/openshift/oadp-operator/pull/1702#discussion_r2033872733

Reply:
I think this ideally should be a buildVersion that @rayfordj can implement downstream like so
```
RUN source ../cachito.env && CGO_ENABLED=1 GOOS=linux go build -a -mod=readonly -ldflags '-X github.com/vmware-tanzu/velero/pkg/buildinfo.Version=v1.12.1-OADP' -tags strictfipsruntime -o ./bin/velero ./cmd/velero
```
[for v1.12.1 OADP](https://catalog.redhat.com/software/containers/oadp/oadp-velero-rhel9/64510133c1647dae8134a158?architecture=amd64&image=655537958f9e90cbffe3e674&container-tabs=dockerfile)

Reply:
eventually anyways.